features/admin.workflow-models.v1 /api/workflow-models.ts

```diff
@@ -0,0 +1,143 @@
+import { AsgardeoSPAClient } from "@asgardeo/auth-react";
+import { store } from "@wso2is/admin.core.v1/store";
+import { HttpMethods } from "@wso2is/core/models";
+import { WorkflowListItemInterface, WorkflowDetails, WorkflowTemplate, WorkflowModelPayload } from "../models/workflow-models";
+import useRequest, { RequestConfigInterface, RequestErrorInterface, RequestResultInterface } from "@wso2is/admin.core.v1/hooks/use-request";
+import { AxiosError, AxiosResponse } from "axios";
+
+/**
+ * Get an axios instance.
+ *
+ */
+const httpClient = AsgardeoSPAClient.getInstance()
+    .httpRequest.bind(AsgardeoSPAClient.getInstance())
+    .bind(AsgardeoSPAClient.getInstance());
+
+export const fetchWorkflows = < Data = WorkflowListItemInterface[], Error = RequestErrorInterface>(
```

Comment:
Since this is a custom hook, let's rename as follows. also let's add a doc comment too.

```suggestion
export const useGetWorkflows = < Data = WorkflowListItemInterface[], Error = RequestErrorInterface>(
```

Reply:
Addressed in 
commit 68d7192da25c8e0e2360d8e39f7cb26af93db237